app/models/documentation/page.rb

```diff
@@ -38,5 +38,16 @@ def was_home_page?
     def previous_workspace
       Workspace.find(documentation_workspace_id_was)
     end
+
+    def latest_update_date
+      updated_at.strftime('%Y-%m-%d %I:%M%p')
+    end
+
+    def title_with_ancestors
+      return title if parent.blank?
+
+      parents = ancestors.reverse.map(&:title).join(' > ')
```

Comment:
como están los N+1 queries con este cambio?

Reply:
Just found the N+1 Query... It's fixed on my last [commit](https://github.com/Grupo-AFAL/documentation/pull/47/commits/d84dc77d9563ab3d9e88ab603450327c90622f93).